### ISSUE TITLE
Rotate not working

### ISSUE THREAD

**Issue by JasonTheAdams** [FortAwesome/react-fontawesome#80]
Tried rotating with the following and it didn't work:
```jsx
<FontAwesomeIcon className="icon" icon={icon} rotate={90} />
```

Switched to this and it worked:
```jsx
<FontAwesomeIcon className="icon" icon={icon} transform={{ rotate: 90 }} />
```

**Comment by robmadole** [comment]
Thanks @JasonTheAdams. We'll take a look.

**Comment by jaotors** [comment]
i think instead of `rotate` it is `rotation`

**Comment by JasonTheAdams** [comment]
@jaotors Ah, yes indeed. Just created a PR to fix this in the documentation.

**Comment by JasonTheAdams** [comment]
Whoops! Ignore the first PR notification. Accidentally made the PR on my fork. 😅

**Comment by robmadole** [comment]
Merged #89
